quipucords/fingerprinter/runner.py

```diff
@@ -138,7 +139,7 @@ def execute_task(self, manager_interrupt):
             )
             raise error
 
-    def _process_details_report(self, manager_interrupt, report):  # noqa: PLR0915
+    def _process_details_report(self, manager_interrupt, report):  # noqa: PLR0915, C901, PLR0912
```

Comment:
I don't like shushing C901. That's a good check IMO. I only found ok turning it off for legacy functions that were already violating it.

Reply:
OTOH, I think this method was already pretty bad before. These new conditionals are not what really makes this a mess...

Reply:
I agree. Normally when I see C901 and related complexity alerts, that motivates me to refactor the code. In this specific case, though, I want to touch the code as little as possible. This code was just barely on the edge of hitting the complexity alert before, and my small bypass conditions pushed it over the edge. 😕